modular_skyrat/modules/liquids/code/liquid_systems/liquid_effect.dm

```diff
@@ -35,6 +35,15 @@
 
 	var/no_effects = FALSE
 
+	/// State-specific message chunks for examine_turf()
+	var/static/list/liquid_state_messages = list(
+		"[LIQUID_STATE_PUDDLE]" = "has a puddle of",
+		"[LIQUID_STATE_ANKLES]" = "is ankle-deep with",
+		"[LIQUID_STATE_WAIST]" = "is waist-deep with",
+		"[LIQUID_STATE_SHOULDERS]" = "is shoulder-deep with",
+		"[LIQUID_STATE_FULLTILE]" = "is full of",
```

Comment:
I don't exactly know how to phrase these better, but those descriptions sound a bit odd to me. We can discuss them in the coding channels if you'd like, I don't mind brainstorming some better ones.